about.html

```diff
@@ -0,0 +1,292 @@
+										<a class="nav-link text-light" href="#">Program</a>
+									</li>
+									<li class="nav-item mb-3">
+										<a class="nav-link text-light" href="#">Sponsor</a>
+									</li>
+									<li class="nav-item mb-3">
+										<a class="nav-link text-light" href="#">Log out</a>
+									</li>
+									<li class="nav-item mb-3 border border-5 border-danger p-3">
+										<a class="nav-link text-center text-danger" href="#"
+											>Our legacy</a
+										>
+									</li>
+								</ul>
+							</div>
+						</div>
+						--
+					</div>
+				</nav>
+			</div>
+				<!-- desktop navigation -->
+				<div id="desktop-toolbar">
+					<div class="container-fluid bg-dark">
+						<ul class="nav justify-content-end">
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#"
+									><i class="fa-brands fa-facebook-f"></i
+								></a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">
+									<i class="fa-brands fa-twitter"></i>
+								</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">English</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">My Page</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">Log out</a>
+							</li>
+						</ul>
+					</div>
+
+					<nav class="navbar navbar-expand-sm">
+						<div class="container-fluid bg-white">
+							<a class="navbar-brand text-dark" href="index.html"
+								><img
+									class="victoria-logo"
+									src="./asset/images/victoria-university-logo.png"
+									alt="Victoria-university-logo" />
+							</a>
+
+							<ul
+								class="navbar-nav conatiner-fluid d-flex justify-content-between align-items-center">
+								
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="./index.html">Home</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a
+										class="nav-link text-dark"
+										id="desktop-nav"
+										href="./about.html"
+										>About</a
+									>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Program</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Sponsor</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Log out</a>
+								</li>
+								<li class="nav-item mx-4 border border-5 border-danger">
+									<a class="nav-link text-center text-danger" href="#"
+										>Our legacy</a
+									>
+								</li>
+							</ul>
+						</div>
+					</nav>
+				</div>
+			</header>
+
+			<main>
+				<!--Intro section -->
+				<section >
+					<div class="hero-container py-3">
+						<h1 class="h1 text-danger text-center mb-3 hero-about-content">
```

Comment:
[REQUIRED] Kindly ensure that your pages have only a single h1 tag which contains the main text of that page